dist/constants.json

```diff
@@ -5883,931 +5883,931 @@
       },
       {
         "value": "405GA",
-        "label": "BENNINGTON VERMONT OUTREACH CLINIC"
+        "label": "Bennington VA Clinic"
       },
       {
         "value": "405HA",
-        "label": "BURLINGTON LAKESIDE CBOC"
+        "label": "Burlington Lakeside VA Clinic"
       },
       {
         "value": "405HB",
         "label": "MONTPELIER OUTREACH CLINIC"
       },
       {
         "value": "405HF",
-        "label": "RUTLAND VERMONT OUTREACH CLINIC"
+        "label": "Rutland VA Clinic"
       },
       {
         "value": "405GC",
-        "label": "BRATTLEBORO CBOC"
+        "label": "Brattleboro VA Clinic"
       },
       {
         "value": "405",
-        "label": "WHITE RIVER JCT VAMROC"
+        "label": "White River Junction Regional Benefit Office"
```

Comment:
Let's confirm this name change.

This seems to be referencing the Benefits Office, not the Medical Facility.